x-pack/metricbeat/module/aws/cloudwatch/_meta/fields.yml

```diff
@@ -0,0 +1,5 @@
+- name: cloudwatch
+  type: group
+  description: >
+    `cloudwatch` contains the metrics that were scraped from AWS CloudWatch which contains monitoring metrics sent by different namespaces.
+  release: beta
```

Comment:
We should aim for a good dynamic mapping here, for instance:

`cloudwatch.metrics.*` for metrics (double)
`cloudwatch.labels.*` for labels (keyword)
`cloudwatch.namespace` (keyword)

You can get some inspiration from https://github.com/elastic/beats/blob/master/metricbeat/module/prometheus/_meta/fields.yml

Reply:
Yes, done. Thank you @exekias ! I named `cloudwatch.dimensions.*` instead of `labels` because these dimensions are from `MetricDataQueries` in cloudwatch api `GetMetricData`: https://aws.amazon.com/premiumsupport/knowledge-center/cloudwatch-getmetricdata-api/